docs: note descending default sort for numeric heat map Y axis (#7051) (#7086)
elastic/docs-content: explore-analyze/visualize/charts/heat-map-charts.md

**Sort order** {applies_to}`serverless: ga` {applies_to}`stack: ga 9.4`
:   Control the sort order of the vertical axis values:
- **Unsorted**: Use the default sort order (default).

   {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` When the vertical axis uses numeric data, the unsorted default sorts rows in descending numeric order. Axes that use other data types, such as strings, are unaffected: their rows appear in the order the data is returned. You can still set the sort order manually.

>    

Author: ```suggestion
    - **Unsorted**: Use the default sort order (default).

       {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` When the vertical axis uses numeric data, the unsorted default sorts rows in descending numeric order. Axes that use other data types, such as strings, are unaffected: their rows appear in the order the data is returned. You can still set the sort order manually.
   
```

**Sort order** {applies_to}`serverless: ga` {applies_to}`stack: ga 9.4`
:   Control the sort order of the vertical axis values:
    - **Unsorted**: Use the default sort order (default).

>         {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` When the vertical axis uses numeric data, the unsorted default sorts rows in descending numeric order. Axes that use other data types, such as strings, are unaffected: their rows appear in the order the data is returned. You can still set the sort order manually.

Author: ```suggestion
:   Control the sort order of the vertical axis values:
- **Unsorted**: Use the default sort order (default).

   {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` When the vertical axis uses numeric data, the unsorted default sorts rows in descending numeric order. Axes that use other data types, such as strings, are unaffected: their rows appear in the order the data is returned. You can still set the sort order manually.
   
```

**Sort order** {applies_to}`serverless: ga` {applies_to}`stack: ga 9.4`
:   Control the sort order of the vertical axis values:
    - **Unsorted**: Use the default sort order (default).

>        {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` When the vertical axis uses numeric data, the unsorted default sorts rows in descending numeric order. Axes that use other data types, such as strings, are unaffected: their rows appear in the order the data is returned. You can still set the sort order manually.

Author: ```suggestion
       {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` When the vertical axis uses numeric data, descending numeric order is used. Axes that use other data types, such as strings, are unaffected: their rows appear in the order the data is returned. You can still set the sort order manually.
```
